src/main/java/com/exasol/containers/ExasolContainer.java

```diff
@@ -992,14 +1004,23 @@ DockerAccess createDockerAccess() {
     private SshKeys getSshKeys() {
         try {
             return SshKeys.builder() //
-                    .privateKey(Paths.get("target/id_rsa")) //
-                    .publicKey(Paths.get("target/id_rsa.pub")) //
+                    .privateKey(getTemporaryCredentialsDirectory().resolve("id_rsa")) //
+                    .publicKey(getTemporaryCredentialsDirectory().resolve("id_rsa.pub")) //
                     .build();
         } catch (final JSchException | IOException exception) {
             throw new ExasolContainerInitializationException("Could not create SSH key pair", exception);
         }
     }
 
+    /**
+     * Get the path to the directory where the Exasol container stores temporary credentials.
+     *
+     * @return directory for temporary credentials
+     */
+    public Path getTemporaryCredentialsDirectory() {
```

Comment:
```suggestion
    private Path getTemporaryCredentialsDirectory() {
```
I think it's not necessary to make this public (yet). Maybe in the future if it's required.